util/dataframe_ops.py

```diff
@@ -273,40 +282,61 @@ def parse_check(tolerance_file_name, input_file_ref, input_file_cur, factor):
     """
     df_tol = parse_probtest_csv(tolerance_file_name, index_col=[0, 1])
 
-    logger.info("applying a factor of %s to the spread", factor)
+    # logger.info("applying a factor of %s to the spread", factor)
     df_tol *= factor
 
     df_ref = parse_probtest_csv(input_file_ref, index_col=[0, 1, 2])
     df_cur = parse_probtest_csv(input_file_cur, index_col=[0, 1, 2])
 
-    logger.info(
-        "checking %s against %s using tolerances from %s",
-        input_file_cur,
-        input_file_ref,
-        tolerance_file_name,
-    )
+    # logger.info(
+    #     "checking %s against %s using tolerances from %s",
+    #     input_file_cur,
+    #     input_file_ref,
+    #     tolerance_file_name,
+    # )
 
     return df_tol, df_ref, df_cur
 
 
 def check_stats_file_with_tolerances(
-    tolerance_file_name, input_file_ref, input_file_cur, factor
+    tolerance_file_name, input_file_ref, input_file_cur, factor, type_f="stats"
```

Comment:
I think we could make the assumptions that stats-files and fof files, start with stats and fof, respectively, based on that we could save this function parameter.
This assumption needs to be clearly documented and ideally checked.